### PR TITLE
Add delete disaster functionality

### PR DESCRIPTION
**Pull request by juliexxia**

### CODE REVIEW

File: cypress/integration/unit_tests/add_disaster_test.js [hunk 1]
```diff
@@ -184,6 +184,35 @@ describe('Unit tests for add_disaster page', () => {
...
+          expect(doc.exists).to.be.true;
+          return cy.window().then((win) => deleteDisaster(win));
+        })
+        .then(() => {
+          return getFirestoreRoot()
+              .collection('disaster-metadata')
+              .doc(id)
+              .get();
+        })
+        .then((doc) => {
```
Comment (janakdr) [inline cypress/integration/unit_tests/add_disaster_test.js]: nit: remove braces from one-liner

File: cypress/integration/unit_tests/add_disaster_test.js [hunk 2]
```diff
@@ -184,6 +184,35 @@ describe('Unit tests for add_disaster page', () => {
...
         .then((success) => expect(success).to.be.true);
   });
+
+  it('deletes a disaster', () => {
+    cy.on('window:confirm', () => true);
+
+    const id = '2002-winter';
+    const states = ['DN, WF'];
+
+    cy.document()
```
Comment (janakdr) [inline cypress/integration/unit_tests/add_disaster_test.js]: why do you need this cy.document()?

Comment (juliexxia) [reply cypress/integration/unit_tests/add_disaster_test.js]: Ah sorry, leftover from figuring out how to get the window to work. Removed.

File: cypress/integration/unit_tests/add_disaster_test.js (hunk 1 above)
Comment (juliexxia) [reply cypress/integration/unit_tests/add_disaster_test.js]: Done.

File: cypress/integration/unit_tests/add_disaster_test.js [hunk 3]
```diff
@@ -184,6 +184,32 @@ describe('Unit tests for add_disaster page', () => {
...
   });
+
+  it('deletes a disaster', () => {
+    cy.on('window:confirm', () => true);
+
+    const id = '2002-winter';
+    const states = ['DN, WF'];
+
+    cy.wrap(writeNewDisaster(id, states))
+        .then(() => {
```
Comment (janakdr) [inline cypress/integration/unit_tests/add_disaster_test.js]: Nit: you can remove braces from this and below

File: docs/import/add_disaster.js [hunk 4]
```diff
@@ -257,6 +262,28 @@ function createAssetPickers(states) {
...
+ * @param {Window} globalWindow DOM Window (for test injection)
+ * @return {Promise<void>}
+ */
+function deleteDisaster(globalWindow = window) {
+  const disasterPicker = $('#disaster');
+  const disasterId = disasterPicker.val();
+  if (globalWindow.confirm(
+          'Delete ' + disasterId + '? This action cannot be undone')) {
+    disasters.delete(disasterId);
+    disasterPicker.val(disasterPicker.children().eq(0).val()).trigger('change');
```
Comment (janakdr) [inline docs/import/add_disaster.js]: What is this for? And don't you do this somewhere else too? Extract to a function?

File: cypress/integration/unit_tests/add_disaster_test.js (hunk 3 above)
Comment (juliexxia) [reply cypress/integration/unit_tests/add_disaster_test.js]: Done.

File: docs/import/add_disaster.js (hunk 4 above)
Comment (juliexxia) [reply docs/import/add_disaster.js]: This sets the picker to the most recent disaster once you delete the one you're looking at. I do it in the cancel PR as well (: I'll extract in whichever I submit second if that's alright with you.

File: cypress/integration/unit_tests/add_disaster_test.js (hunk 3 above)
Comment (janakdr) [reply cypress/integration/unit_tests/add_disaster_test.js]: Don't see it?

**Pull request merged**